lib/python/picongpu/pypicongpu/species/operation/densityprofile/foil.py

```diff
@@ -0,0 +1,59 @@
+"""
+This file is part of the PIConGPU.
+Copyright 2023 PIConGPU contributors
+Authors: Kristin Tippey, Brian Edward Marre
+License: GPLv3+
+"""
+
+from .densityprofile import DensityProfile
+from .... import util
```

Comment:
no, must be 4 otherwise we get the following error.

```python
File "/home/marre55/picongpu/lib/python/picongpu/pypicongpu/species/operation/densityprofile/foil.py", line 22, in Foil
    density_si = util.build_typesafe_property(float)
AttributeError: module 'picongpu.pypicongpu.species.util' has no attribute 'build_typesafe_property'
```
We start from picongpu/pypicongpu/species/operation/densityprofile/foil.py
and want the picongpu/pypicongpu/util.py namespace, not the picongpu/pypicongpu/species/util/ namespace therefore we need 4 jumps back up to picongpu/pypicongpu/ and then search for the first util namespace.

And yes I known that possible name collision is less than ideal.